frontend/src/citizen-frontend/calendar/AbsenceModal.tsx

```diff
@@ -153,6 +167,22 @@ export default React.memo(function AbsenceModal({
   const [attendanceAlreadyExistsError, setAttendanceAlreadyExistsError] =
     useState(false)
 
+  const tooManyAbsencesErrors =
+    featureFlags.absenceApplications &&
+    selectedChildren.isValid() &&
+    range.isValid() &&
+    range.value().durationInDays() > tooManyAbsencesErrorThresholdInDays &&
```

Comment:
Pitäiskö tässä laskea vain eskarin toimintapäiviä? Silloin pitäis katsoa viikonloput, arkipyhät, eskarin toimintakaudet ja eskarin loma-ajat. Lisäksi pitäisi kai huomioida jo aiemmin merkityt poissaolot, jotta ei voi kiertää validaatiota vahingossa/tahallaan ilmoittamalla poissaolon pätkissä. Käytännössä tarviis siis tehdä requ bäkkäriin jossa DateSettiä käyttäen lasketaan tarviiko tehdä poissaolohakemus. Ei mikään ihan nopee ja helppo homma, joten tästä vois varmaan viel jutella PO:iden ja Tommin kanssa, että mitä vois pudottaa MVP-scopesta pois, ellet toki oo jo ehtinyt tätä keskustelua käymään ja sopinut jotain? Toi että `tooManyAbsencesErrorThresholdInDays` on 7 eikä 5 saattaa toki nuo viikonloput jo ratkaista.